packages/salesforcedx-vscode-apex-replay-debugger/README.md

```diff
@@ -1,14 +1,108 @@
+
+---
+NOTE: As a beta feature, Apex Replay Debugger is a preview and isn’t part of the “Services” under your master subscription agreement with Salesforce. Use this feature at your sole discretion, and make your purchase decisions only on the basis of generally available products and features. Salesforce doesn’t guarantee general availability of this feature within any particular time frame or at all, and we can discontinue it at any time. This feature is for evaluation purposes only, not for production use. It’s offered as is and isn’t supported, and Salesforce has no liability for any harm or damage arising out of or in connection with it. All restrictions, Salesforce reservation of rights, obligations concerning the Services, and terms for related Non-Salesforce Applications and Content apply equally to your use of this feature. You can provide feedback and suggestions for Apex Replay Debugger in the [Issues section](https://github.com/forcedotcom/salesforcedx-vscode/issues) of the salesforcedx-vscode repository on GitHub.
+
+---
+
+Apex Replay Debugger helps you debug issues in an after-the-fact manner. Your debugging process is a repetition of editing your Apex code, pushing or deploying the code to your org, reproducing the buggy scenario, downloading the resulting debug log, and launching Apex Replay Debugger with that debug log.
 
 ## Prerequisites
+Before you set up Apex Replay Debugger, make sure that you have these essentials.
+
+* **Salesforce CLI**  
+  Before you use Salesforce Extensions for VS Code, [set up Salesforce CLI](https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup). 
+* **A Salesforce DX project**  
+  See [Project Setup](https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_workspace_setup.htm) in the _Salesforce DX Developer Guide_ for details.
+* **An active default scratch org _OR_ a local copy of a debug log from the org whose up-to-date source is in your Salesforce DX project**
+  1. To create a scratch org, you need a Dev Hub. For information on setting up your production org as a Dev Hub, see [Enable Dev Hub in Your Org](https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_enable_devhub.htm) in the _Salesforce DX Setup Guide_.  
+  1. To authorize your Dev Hub, open VS Code’s command palette (Cmd+Shift+P on macOS, or Ctrl+Shift+P on Windows or Linux) and run **SFDX: Authorize a Dev Hub**.  
+  1. To create a default scratch org, run **SFDX: Create a Default Scratch Org**. Then, run **SFDX: Push Source to Default Scratch Org**. For information about scratch orgs, see [Scratch Orgs](https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_scratch_orgs.htm) in the _Salesforce DX Developer Guide_.
+* **[Visual Studio Code](https://code.visualstudio.com/download) v1.17 or later** 
```

Comment:
I believe this needs to be v1.23 or later. It was bumped up recently to prepare for future projects. This applies to people who auto-update their extension or just installed the extension from VS Code Marketplace. Those that don't have the most recent extensions can still use it with older VS Code version.